d3d/SamplerFeedback.md

```diff
@@ -15,7 +15,7 @@ Over time, there has been steady increase in display resolutions whereupon games
 
 The general process of loading texture data on-demand, rather than upfront-all-at-once, is called *texture streaming* in this document, or *streaming* for short.
 
-It makes sense to use streaming in scenarios where only some of a texture's mips are deemed necessary for the scene. When new mips are deemed necessary-- for example, if a new object appears in the scene, or if an object has moved closer into view and requires more detail-- the application may choose to load more-detailed parts of the mip chain. 

```

Comment:
Thanks @siliconvoodoo for contributing this!
Small nit: this change has a bunch of whitespace changes showing up in the diff, is this intended?

Reply:
you're right. unfortunately it's very much not intended. but after I noticed I decided to leave them because future editors will also sometimes have settings that remove those trailing whitespaces.
It would be idea though to separate the change from the whitespace change in 2 commits. I just go lazy because that's an amend + recommit with chunks selection which is a bit cumbersome :(